vite.config.ts

```diff
@@ -3,5 +3,18 @@ import vue from '@vitejs/plugin-vue'
 
 // https://vitejs.dev/config/
 export default defineConfig({
-	plugins: [vue()]
+	root: './src',
+
+	build: {
+		rollupOptions: {
+			maxParallelFileReads: 512
+		},
+
+		outDir: '../dist'
+	},
+	plugins: [
+		vue({
+			isProduction: true
```

Comment:
This should be changed so that isProduction is true if current branch is main. From the way you set up Husky I don't think this will matter but good to follow for convention's sake